src/components/molecules/PostComment.tsx

```diff
@@ -33,30 +34,65 @@ const PostComment: FC<Props> = (props) => {
   const [childrenOpen, setChildrenOpen] = useState<
     Array<{ index: number; value: boolean }>
   >([])
+  const [childrenCommentsArr, setChildrenCommentsArr] =
+    useState<
+      Array<{ parentId: string; contents: Array<ChildrenCommentInfo> }>
```

Comment:
I'll be better to make interface on top of code for reusability. what do you think about it?